gnovm/cmd/gno/test.go

```diff
@@ -308,6 +309,7 @@ func gnoTestPkg(
 		for _, testFile := range filetestFiles {
 			testFileName := filepath.Base(testFile)
 			testName := "file/" + testFileName
+			filter := splitRegexp(runFlag)
```

Comment:
isn't runFlag immutable? why did this line move into the for loop?

Reply:
It's not a bug fix here, I've just moved the `filter` variable assignment here because it's only used in this block.

Reply:
oups indeed ! done here 836b71de